src/context/mod.rs

```diff
@@ -294,6 +294,12 @@ impl<'a> Lock<'a> {
 ///
 /// A context has a lifetime `'a`, which ensures the safety of handles managed by the JS garbage collector. All handles created during the lifetime of a context are kept alive for that duration and cannot outlive the context.
 pub trait Context<'a>: ContextInternal<'a> {
+    /// Get the underlying `napi-env` of the context
+    #[cfg(feature = "napi-1")]
+    fn to_raw_env(&self) -> *mut c_void {
```

Comment:
I think I would favor a more idiomatic name on the public API instead of matching internal naming. What do you think of `as_mut_ptr`?

Additionally, this function should take `&mut self`. Even though the caller still needs to uphold borrow rules, it would *always* be undefined behavior to get the `napi_env` while a mutable reference to `Context` is held.